docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
 
-You can also define user access per environment type (Production, Staging, Development). Each permission level progressively increases access to the environments part of a given environment type:
+### Transfer a project ownership
 
-* **Viewer** - Viewer can view all environments of this type.
-* **Contributor** - Contributor can push code and branch all environments of this type.
-* **Administrator** - Administrator can change settings and execute actions on all environments of this type.
+As an organization owner, or as an organization user with the **manage plans** permission, you can transfer project ownership to a different organization, first invite that organization owner as a **Project admin** and then, submit a support ticket from your organization owner account to ask for the transfer. Transferring a project automatically transfers the upcoming subscription charges to the new owner. Once you transfer ownership, open a [support ticket](https://console.platform.sh/-/users/~/tickets) to request your account deletion.                                                                                                     
 
-{{< note theme="warning" title="Important" >}}
-After you add or remove a user from a project or an environment type, you will need to trigger a redeploy to propagate the access changes to each environment.
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
+## Environment types
 
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
+Platform.sh offers three types of environment groups: Production, Staging, and
+Development. You can assign permissions to each user per environment so for each environment type, users always have the same set of permissions across all environments of the same type.
 
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
+For example, if you assign user1 **Admin** permissions for Development environments,  
+**Contributor** permissions for Staging environments and **Viewer** for the Production environment. User1 effectively has **Admin** permissions to all development environments, **Contributor** permissions to all staging environments and **Viewer** permissions to the Production environment.
 
-## SSH access
+A few things to consider:
 
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
+* Only one environment per project can have Production type.
+* Environments can share the same Staging and Development type.
```

Comment:
This seems to say that an environment can be of both types, which isn't true.

The idea that you can change the type of an environment seems useful as does the idea that you can have multiple Staging (or Development) environments. So including those ideas would help.